bring-it-up/bring-it-up
# [BE#47] Clean up server files

## Description
#### Summary:
- Updated all API-related docs and files and fixed linting errors

#### Changes:
- Removed `route.rest` file
- Updated Postman collection with up-to-date data and endpoints
- Updated `documentation` folder API docs with latest response data
- Fixed linting errors
- Updated PR template to include lint check

#### How to test this PR:
- Please take a look through the changed API doc files to double check they are accurate
- Make sure the ESLint extension in VS Code is installed and working, and that you can run `npm run lint` without errors. To check that the extension is working, make changes in some files that will break linting rules, and the errors should appear in VS Code.
- Import the Postman collection, and all endpoints should be up to date
  
## Checklist

- [x] I ran all the tests and they were successful
